pedpy/__init__.py

```diff
@@ -63,6 +69,7 @@
     PEDPY_ORANGE,
     PEDPY_PETROL,
     PEDPY_RED,
+    plot_acceleration,
```

Comment:
Please add hist also to the ` __all__` list

Reply:
done